### DISCUSSION TITLE
Audio File Test Set

### DISCUSSION THREAD

**Discussion by kcwhite** [openai/whisper#1176] [Q&A] [answered]
Hey @jongwook,

My company is using Whisper and I've been tasked with doing the QA on it. I'd really like to get a diverse set of audio files in different formats to use as a test set. What have you used during development? I'm not talking about the tests in the repo, but the actual audio files.

My dream test set would look something like this:

- All audio file formats supported by Whisper (Supported formats: [‘m4a’, ‘mp3’, ‘webm’, ‘mp4’, ‘mpga’, ‘wav’, ‘mpeg’])
- Different bitrates per codec (16kbps - 320kbps)
- Both constant bitrate and variable bitrate audio files
- File play times longer than the allowed (is there a limit?)
- File sizes larger than allowed
- Files recorded at both low and high volume
- Files with background noise or distortion

**Comment by jongwook** [comment, marked as answer]
[The paper](https://arxiv.org/abs/2212.04356) contains more details on the datasets we used for development. But I suppose you'd need to collect audio files that more closely match the use cases of your company's audience.

**Reply by kcwhite** [reply]
I apologize if I'm missing something, but that paper deals with computer vision, not audio -> text transcription. The datasets mentioned are photos, not audio files.

**Reply by jongwook** [reply]
Sorry! I linked to a wrong paper - it's fixed now.

**Reply by kcwhite** [reply]
Thanks! 

For others who may see this in the future and want to access the datasets:

1. Click the [link Jong Wook provided](https://arxiv.org/abs/2212.04356), which will take you to the abstract page
2. Scroll to the bottom and select the tab, "Code, Data, Media"
3. Toggle the button labeled "Papers with Code" to the on position

That will show you all the datasets, with links, that were used for training.
